web/README.md

```diff
@@ -0,0 +1,295 @@
+# @thorvg/lottie-player
+
+A Lottie Player which uses [ThorVG](https://github.com/thorvg/thorvg) as a renderer, provides web component for easily embedding and playing [Lottie](https://airbnb.io/lottie) animations.
```

Comment:
@hermet I simply added links for what ThorVG and Lottie are, I'm not sure It's good enough.

Please let me know if you have any idea for head of README